integration-test/src/test/java/org/apache/iotdb/relational/it/insertquery/IoTDBInsertQueryIT.java

```diff
@@ -86,7 +86,8 @@ public static void setUpClass() {
         .getConfig()
         .getCommonConfig()
         .setPartitionInterval(1000)
-        .setMemtableSizeThreshold(10000);
+        .setMemtableSizeThreshold(10000)
+        .setSelectIntoInsertTabletPlanRowLimit(1000);
```

Comment:
I think this parameter is stale, insert into query rows in table model, shouldn't be limited by this parameter